COMMUNITY_MEMBERSHIP.md

```diff
@@ -1,35 +1,49 @@
-# Keptn Community membership
-
-**Last updated:** 2021-08-26
+# Contributor Ladder
 
 **Disclaimer:** This document borrows heavily from the [Community Membership of OpenTelemetry](https://github.com/open-telemetry/community/blob/3f44aedd70334999b8a63586cf44f747776bddd3/community-membership.md).
 
-This document outlines the various responsibilities of contributor roles in Keptn. 
-The Keptn project consists of:
-  * one **main project** with the [core services of Keptn](https://github.com/keptn/keptn)
-  * **Keptn-contrib** that contains [integrations and contributions](https://github.com/keptn-contrib) for Keptn
-  * **Keptn-sandbox** that new [contributions to Keptn by the community](https://github.com/keptn-sandbox) and not maintained by Keptn core maintainers 
+**Ownership:** This document is owned by the Governance committee, Technichal committee and community manager [ADD LINK TO GOVERNACE AND TC]().
 
-Responsibilities for the roles in this document are scoped to the *main project* and *Keptn contrib*.
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows. Our project members are happy to help you advance along the contributor ladder.
```

Comment:
I would recommend splitting those bit lines into multiple ones.

Markdown will still see them as one paragraph and render them as a paragraph. But it is easier to maintain if you put eg. one sentence per line.